.github/workflows/docs.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Grab gtk-rs LOGO
         if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/master') || github.event_name == 'release' }}
         run: |
-          sudo dnf install wget -y
+          apt-get install wget -y
```

Comment:
Nowadays you'd usually just use `apt` instead of `apt-get` FWIW

Reply:
Actually, let's move the wget install to the docker image, there's no reason to re-install it everytime here